pkg/virt-controller/services/template.go

```diff
@@ -1326,21 +1327,28 @@ func (t *templateService) RenderHotplugAttachmentPodTemplate(volume *v1.Volume,
 	return pod, nil
 }
 
-func getRequiredCapabilities(vmi *v1.VirtualMachineInstance) []k8sv1.Capability {
-	res := []k8sv1.Capability{}
+func getRequiredCapabilities(vmi *v1.VirtualMachineInstance, config *virtconfig.ClusterConfig) []k8sv1.Capability {
+	capabilities := []k8sv1.Capability{}
```

Comment:
Please do this change in a separate commit, it is confusing to see renaming changes with the other changes.

Reply:
done